<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>songSettingsWindow</class>
 <widget class="QWidget" name="songSettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>490</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="SettingsScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="songSettings">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>405</width>
        <height>1388</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="ffmpegSettings">
         <property name="title">
          <string>FFmpeg settings (song)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="CoverArtDisplay" name="coverArt">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Cover art</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="coverArtButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Choose cover art...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Background color:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="backgroundColor"/>
          </item>
          <item>
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Video width:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="videoWidth">
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>Video height:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="videoHeight">
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Output file directory:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fileOutputDir"/>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Output file name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fileOutputName"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Render command:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="FileComboBox" name="commandName"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="albumPlaylistLabel">
         <property name="text">
          <string>Render album as:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="albumPlaylist"/>
       </item>
       <item>
        <widget class="QGroupBox" name="ffmpegSettingsAlbum">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>FFmpeg settings (album)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Output file directory:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fileOutputDirAlbum"/>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Output file name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fileOutputNameAlbum"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Concatenation command name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="FileComboBox" name="concatCommandName"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="SettingCheckBox" name="uploadYouTube">
         <property name="text">
          <string>Upload to YouTube?</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="youtubeSettings">
         <property name="title">
          <string>YouTube settings (song)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Video title:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="videoTitle"/>
          </item>
          <item>
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Video description:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="videoDescription"/>
          </item>
          <item>
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Video tags:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="videoTags">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Playlist names (one per line):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="playlistName"/>
          </item>
          <item>
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Visibility:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="videoVisibility"/>
          </item>
          <item>
           <widget class="SettingCheckBox" name="notifySubs">
            <property name="text">
             <string>Notify subscribers?</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="youtubeSettingsAlbum">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>YouTube settings (album)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_60">
          <item>
           <widget class="QLabel" name="label_200">
            <property name="text">
             <string>Video title:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="videoTitleAlbum"/>
          </item>
          <item>
           <widget class="QLabel" name="label_210">
            <property name="text">
             <string>Video description:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="videoDescriptionAlbum"/>
          </item>
          <item>
           <widget class="QLabel" name="label_220">
            <property name="text">
             <string>Video tags:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="videoTagsAlbum">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>Visibility:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="videoVisibilityAlbum"/>
          </item>
          <item>
           <widget class="SettingCheckBox" name="notifySubsAlbum">
            <property name="text">
             <string>Notify subscribers?</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="songSettingsButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SettingCheckBox</class>
   <extends>QCheckBox</extends>
   <header>setting_check_box.h</header>
  </customwidget>
  <customwidget>
   <class>CoverArtDisplay</class>
   <extends>QLabel</extends>
   <header>cover_art_display.h</header>
  </customwidget>
  <customwidget>
   <class>SettingsScrollArea</class>
   <extends>QScrollArea</extends>
   <header>settings_scroll_area.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileComboBox</class>
   <extends>QComboBox</extends>
   <header>file_combo_box.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
